system_launch/package.xml

```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>system_launch</name>
   <version>0.1.0</version>
   <description>The system_launch package</description>
 
   <maintainer email="kenji.miyake@tier4.jp">Kenji Miyake</maintainer>
-  <license>Apache 2</license>
+  <license>Apache License 2.0</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
 
```

Comment:
please add `<exec_depend>` for all those packages whose launch file is used in `system.launch.xml`

Reply:
And when it's done, see if you can launch everything successfully